packages/notification-center/src/store/novu-theme.context.ts

```diff
@@ -47,6 +47,7 @@ export interface IThemeHeader {
 
 export interface IThemePopover {
   arrowColor?: string;
+  tabBorderColor?: string;
```

Comment:
@aggmoulik @p-fernandez What do you think about adding this to the IThemeHeader instead of the IThemePopover? As the unseen badge color is also defined in the header
And also because the popover feels more fitting for more general settings of the actual pop up

Reply:
Good suggestion. Agree!